src/core/Form/Radiobutton/Radiobutton.baseStyles.tsx

```diff
@@ -0,0 +1,159 @@
+import { css } from 'styled-components';
+import { withSuomifiTheme, TokensAndTheme } from '../../theme';
+import { disabledCursor } from '../../../components/utils/css';
+import { element, font } from '../../theme/reset';
+import { focus } from '../../theme/utils/focus';
+
+/* stylelint-disable no-descending-specificity */
```

Comment:
Had to add this `no-descending-specificity` after all different ways to prevent the error. 
If anyone knows the correct way to fix this and current functionality stays same and styles are visually intact, I would like to learn it.

Reply:
I think we had similar issues with other components. This rule does not provide much value with css-in-js solution in my opinion.